src/Lotto/Ticket.js

```diff
@@ -0,0 +1,26 @@
+import { Random } from '@woowacourse/mission-utils';
+
+class Ticket {
+  #tickets = [];
+
+  constructor(money) {
+    for (let i = 0; i < Math.floor(this.#initTicketCount(money)); i += 1) {
+      this.#tickets.push(this.#purchaseTicket());
+    }
+  }
```

Comment:
```suggestion
constructor(money) {
    const ticketCount = Math.floor(this.#initTicketCount(money));
    this.#tickets = Array.from({ length: ticketCount }, () => this.#purchaseTicket());
}

```
for문 대신 이런 고차함수로 작성하는 건 어떨까요? 
고차 함수는 코드의 의도를 명확하게 나타낼 수 있어 읽기 쉬운 코드를 만드는데 도움을 주는 이점이 있다고 해요.

Reply:
너무 좋은 피드백인것 같습니다 감사합니다 
제공해주신 코드가 훨씬 가독성이 좋은 것 같습니다